test/versioned/aws-sdk-v3/bedrock-chat-completions.tap.js

```diff
@@ -135,25 +135,30 @@ tap.afterEach(async (t) => {
       const api = helper.getAgentApi()
       helper.runInTransaction(agent, async (tx) => {
         api.addCustomAttribute('llm.conversation_id', 'convo-id')
-        await client.send(command)
-        const events = agent.customEventAggregator.events.toArray()
-        t.equal(events.length, 3)
-        const chatSummary = events.filter(([{ type }]) => type === 'LlmChatCompletionSummary')[0]
-        const chatMsgs = events.filter(([{ type }]) => type === 'LlmChatCompletionMessage')
-
-        t.llmMessages({
-          modelId,
-          prompt,
-          resContent: '42',
-          tx,
-          expectedId: modelId.includes('ai21') || modelId.includes('cohere') ? '1234' : null,
-          chatMsgs
+        api.withLlmCustomAttributes({ 'llm.contextAttribute': 'someValue' }, async () => {
```

Comment:
As with the Langchain tests, please do not overload this existing test. Extract the new functionality verification into it's own test and restore the intention of this original test.

Reply:
In this case, test is checking if custom events being created properly which seems to be a good place to incorporate the check to support custom attributes.

I can make a copy of that if the team feels strongly about that.

Reply:
_All_ LLM events are "custom events" due to the design of the system. The title of this test is on line 128: "properly create the LlmChatCompletionMessage(s) and LlmChatCompletionSummary events". So I expect to see those two things covered in this test, and nothing else.

It's easier to reason about a test if it only does the thing it is advertised to be doing.